.vscode/settings.json

```diff
@@ -1,42 +1,53 @@
 {
-    "editor.fontFamily": "Fira Code, Consolas, 'Courier New', monospace",
-    "editor.fontLigatures": true,
-    "editor.snippetSuggestions": "top",
-    "terminal.integrated.fontFamily": "Consolas",
-    "workbench.colorTheme": "Dracula Soft",
-    "cSpell.ignoreWords": [
-        "aichner",
-        "eupl",
-        "mdbreact",       
-        "spdx",
-        "werbeagentur",
-        "fontawesome",
-        "fortawesome",
-        "reactjs",
-        "react.js"
-    ],
-    "cSpell.enabledLanguageIds": [
-        "html",
-        "go",
-        "javascript",
-        "javascriptreact",
-        "markdown",
-        "php",
-        "plaintext",
-        "text",
-        "typescript",
-        "typescriptreact",
-        "yml"
-    ],
-    "cSpell.showStatus": true,
-    "cSpell.ignorePaths": [
-        "node_modules"
-    ],
-    "cSpell.words": [
-        "browserslist",
-        "homescreen",
-        "precached",
-        "renamer",
-        "webfonts"
-    ]
+  "editor.fontFamily": "Fira Code, Consolas, 'Courier New', monospace",
```

Comment:
Add these settings:
 - "prettier.trailingComma": "es5",
 - "prettier.arrowParens": "always",

Reply:
Just copy the .prettierrc file from intel. This will update the settings for prettier.

Reply:
I would suggest to implement this like intel does:
https://github.com/snek-at/intel/pull/77#discussion_r436252307